.github/workflows/get-covid-clade-counts.yaml

```diff
@@ -4,6 +4,11 @@ on:
   schedule:
     - cron: "37 04 * * 1" # 4:37 AM UTC every Mon
   workflow_dispatch:
+    inputs:
```

Comment:
Allowing someone to pass in YYYY-MM-DD when running this workflow manually makes it easier to recover in the event of an error.

In this case, I think we can pass in the dates of the failed runs:

* `2024-10-25`: not the current date, so script will [set the time to 11:59 PM](https://github.com/reichlab/get-covid-clade-counts/blob/main/get_covid_clade_counts.py#L64) (i.e., will include info from the 10/25 Nextstrain run)
* `2024-12-02`: is today, so script will use the action's run time as the as_of timestamp